src/protocol.vala

```diff
@@ -453,11 +504,22 @@ namespace LanguageServer {
         TypeParameter = 25
     }
 
+    class CompletionClientCapabilities : Object {
```

Comment:
To keep the purpose of this merge commit clear, please don't change anything else that is not strictly pertinent to the issue.